### PR TITLE
[ZT] Clean up user logs

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/policies/gateway/identity-selectors.md [hunk 1]
```diff
@@ -19,15 +19,7 @@ If you add or remove a user from a group in your IdP, Gateway will not detect th
...
-### View a user's identity
-
-To view the identity that Gateway will use when evaluating policies:
-
-1. In [Zero Trust](https://one.dash.cloudflare.com/), go to **My Team** > **Users**. This page lists the users who have authenticated to the WARP client.
-2. Select a user.
-3. Under **User Registry identity**, select the user name.
-
-This page shows the information reported by the IdP when the user registered the WARP client.
+To view the identity that Gateway will use when evaluating policies, refer to their [User Registry identity](/cloudflare-one/insights/logs/users/).
```
Comment (ranbel) [inline content/cloudflare-one/policies/gateway/identity-selectors.md]:
```suggestion
To view the identity that Gateway will use when evaluating policies, check the [user registry](/cloudflare-one/insights/logs/users/).
```

File: content/cloudflare-one/insights/logs/users.md [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
-
-- Their user details
-- Devices they have connected to Cloudflare Zero Trust
-- Their recent logins
-
-Select **View all activity** to see a full list of the user’s login events.
+- **User Registry identity**: Select the user's name to view their most recent user identity. Gateway will use this identity when evaluating [firewall, egress, and resolver policies](/cloudflare-one/policies/gateway/). This identity is refreshed when the user re-authenticates WARP, logs into an Access application, or has their IdP group membership updated via [SCIM provisioning](/cloudflare-one/identity/users/scim/). To track how the user's identity has changed over time, go to the **Audit logs** tab.
+- **Session identities**: The user's identity when they authenticated to a specific Access application. Learn more about [Access session management](/cloudflare-one/identity/users/session-management/).
+- **Devices**: Devices registered to the user via WARP.
+- **Recent activities**: Login events from the past 30(?) days.
```
Comment (ranbel) [inline content/cloudflare-one/insights/logs/users.md]: @kennyj42 What's the timespan for our Recent Activities log?

**Comment by ranbel** [comment]
@kennyj42 for review

File: content/cloudflare-one/insights/logs/users.md (hunk 2 above)
Comment (ranbel) [reply content/cloudflare-one/insights/logs/users.md]: I figured this out by looking at the API calls.

**Review by maxvp — APPROVED** [review]

File: content/cloudflare-one/insights/logs/users.md [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
 # User logs
 
-Cloudflare Zero Trust allows you to consult a comprehensive list of users who have authenticated to Cloudflare Zero Trust. For each user that logged in, you can see their name, their email address, and whether they’re actively utilizing a seat in both Access and Gateway.
+User logs show a list of all users who have authenticated to Cloudflare Zero Trust. For each user who has logged in, you can view their enrolled devices, login history, seat usage, and identity used for policy enforcement.
 
-To see your user list, go to **My Team** > **Users**.
+## View user logs
 
-## Revoke user sessions
+In [Zero Trust](https://one.dash.cloudflare.com/), go to **My Team** > **Users**. This page lists all users who have registered the WARP client or authenticated to a Cloudflare Access application. You can select a user's name to view detailed logs, [revoke their session](/cloudflare-one/identity/users/session-management/#revoke-user-sessions), or [remove their seat](/cloudflare-one/identity/users/seat-management/).
```
Comment (kennyj42) [inline content/cloudflare-one/insights/logs/users.md:13]: should we mention that removing a seat also revokes sessions? or does that just need to be in the seat management link

File: content/cloudflare-one/insights/logs/users.md [hunk 4]
```diff
@@ -1,25 +1,20 @@
...
-
-Selecting the View button for any user will show you details for that user, such as:
-
-- Their user details
-- Devices they have connected to Cloudflare Zero Trust
-- Their recent logins
-
-Select **View all activity** to see a full list of the user’s login events.
+- **User Registry identity**: Select the user's name to view their last seen identity. This identity is refreshed when the user re-authenticates WARP, logs into an Access application, or has their IdP group membership updated via {{<glossary-tooltip term_id="SCIM" link="/cloudflare-one/identity/users/scim/">}}SCIM provisioning{{</glossary-tooltip>}}. To track how the user's identity has changed over time, go to the **Audit logs** tab.
+- **Session identities**: The user's identity when they authenticated to a specific Access application. Learn more about [Access session management](/cloudflare-one/identity/users/session-management/).
```
Comment (kennyj42) [inline content/cloudflare-one/insights/logs/users.md]: Idk if the session management link is the right spot. This is a pretty specific concept here, it doesn't have to do with actual session lengths.

**Review by kennyj42 — APPROVED** [review]

File: content/cloudflare-one/insights/logs/users.md (hunk 4 above)
Comment (ranbel) [reply content/cloudflare-one/insights/logs/users.md]: The session identities show an "expiration time". What determines that expiration time, if it has nothing to do with the session length?

File: content/cloudflare-one/insights/logs/users.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/insights/logs/users.md:13]: I think it's fine to leave it in the seat management link. Also, the dash clearly says "Remove a user will revoke their sessions".

File: content/cloudflare-one/insights/logs/users.md (hunk 4 above)
Comment (kennyj42) [reply content/cloudflare-one/insights/logs/users.md]: yea, that's fair. they are session with those defined expirations. we can leave it

**Pull request merged**